src/request.ts

```diff
@@ -95,31 +101,45 @@ export async function getHttpAgents(
   return agents;
 }
 
+export function resetAxios() {
+  _axiosInstances = null;
+}
```

Comment:
That is only needed for the test environment to reset the internal state of this module